pkg/chartutil/create.go

```diff
@@ -500,7 +500,18 @@ spec:
     - name: wget
       image: busybox
       command: ['wget']
-      args: ['{{ include "<CHARTNAME>.fullname" . }}:{{ .Values.service.port }}']
+      args: ['-O', '/dev/null', '{{ include "<CHARTNAME>.fullname" . }}:{{ .Values.service.port }}']
+      securityContext:
+        runAsUser: 1000
+        runAsNonRoot: true
+        readOnlyRootFilesystem: true
+      resources:
+        limits:
+          memory: 60Mi
+          cpu: 200m
+        requests:
+          memory: 10Mi
+          cpu: 10m
```

Comment:
We usually recommend setting the resources to be opt-in. The reason for this is that they can vary widely. For example, take a PostgreSQL instance. What it needs depends on how much data or connections are being made to it. This is difficult for the chart author to determine.

Why set the resources here and if it's needed, is there a better way to handle this for the application operator (chart installer)?